packages/components/src/components/tile/_tile.scss

```
@@ -52,6 +52,10 @@
       background: $layer-hover;
     }
   }
+  .#{$prefix}--tile-clickable tbody tr:hover td .#{$prefix}--link,
+  .#{$prefix}--tile-clickable tbody tr:hover th .#{$prefix}--link {
```

I think we just need something like

```suggestion
  .#{$prefix}--tile--expandable .#{$prefix}--link {
```